Lib/glyphsLib/builder/sources.py

```diff
@@ -30,6 +32,7 @@ def to_designspace_sources(self):
     regular_master = get_regular_master(self.font)
     for master in self.font.masters:
         _to_designspace_source(self, master, (master is regular_master))
+    _to_designspace_source_layer(self)
```

Comment:
So this mean we are always appending the layerName-sources at the end of the sources list in the designspace, correct?
I would expect that the original ordering of the master and non-master layers (associated with the latter) is kept instead.
E.g., a font has two masters Light and Black, and two intermediate "brace" layers for Regular and Medium; then the generated designspace would have the sources written out as: Light, Regular, Medium and Bold. However, according to the current code the order is Light, Bold, and then the brace layers Regular and Medium.

Reply:
another reason to keep the order is when going back from DS to glyphs, to be able to determine to which master a layer source is associated with (i.e. the one immediately preceding it)

Reply:
Brace layers are now inserted after their associated master and sorted by location.